packages/app/src/runner/useRunnerStyle.ts

```diff
@@ -33,8 +33,10 @@ export const useRunnerStyle = ({
   const { width: windowWidth, height: windowHeight } = useWindowSize()
 
   const containerWidth = computed(() => {
+    const { isSpecsListOpen } = useRunnerUiStore()
+
     const miscBorders = 4
-    let nonAutWidth = reporterWidth.value + specListWidth.value + (autMargin * 2) + miscBorders
+    let nonAutWidth = reporterWidth.value + (isSpecsListOpen ? specListWidth.value : 0) + (autMargin * 2) + miscBorders
```

Comment:
On the next line we have

```ts
if (window.__CYPRESS_MODE__ !== 'run') {
      nonAutWidth += collapsedNavBarWidth
```

Should we move this to the `nonAutWidth` assignment, just so we have it all in one place? eg

```ts
const nonAutWidth = reporterWidth.value + (isSpecsListOpen ? specListWidth.value : 0) + (autMargin * 2) + miscBorders + (window.__CYPRESS_MODE__ === 'open' ? collapsedNavBarWidth : 0)
```